charts/honeycomb/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Honeycomb Kubernetes Agent Helm Chart Changelog
 
+## Honeycomb Kubernetes Agent v1.8.0
+ - feat: Added support for additional env variable settings (#242)
```

Comment:
Since this is a contributed piece, we should use the creator's name, linked to the creator's github, please

Reply:
Okay, I added it.